plugins/woocommerce/client/blocks/tests/e2e/tests/mini-cart/mini-cart-block.merchant.block_theme.spec.ts

```diff
@@ -3,16 +3,19 @@
  */
 import { test, expect, BlockData } from '@woocommerce/e2e-utils';
 
+/**
+ * Internal dependencies
+ */
+import { blockData as baseBlockData } from './utils';
+
 const blockData: BlockData = {
-	name: 'Mini-Cart',
-	slug: 'woocommerce/mini-cart',
-	mainClass: '.wc-block-minicart',
+	...baseBlockData,
 	selectors: {
+		...baseBlockData.selectors,
 		editor: {
 			block: '.wp-block-woocommerce-mini-cart',
 			insertButton: "//button//span[text()='Mini-Cart']",
 		},
-		frontend: {},
 	},
 };
```

Comment:
Not related to this PR, but since `blockData` is already defined in `utils.ts`, I decided to reuse it to avoid code duplication.